### PR TITLE
[BigQuery] Standardize starter queries

### PR DESCRIPTION
**Pull request by CynthiaXJia**
Makes boilerplate query through context menu available for all resources (previously unavailable for models and views), centralizes starter query text to the utils folder.

### CODE REVIEW

File: jupyterlab_bigquery/src/utils/starter_queries.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+export function getStarterQuery(
+  type: 'MODEL' | 'TABLE' | 'VIEW',
```
Comment (cbwilkes) [inline jupyterlab_bigquery/src/utils/starter_queries.ts:2]: Can this be turned into a type:

```
type QueryType = 'MODEL' | 'TABLE' | 'VIEW';
```

File: jupyterlab_bigquery/jupyterlab_bigquery/list_items_handler/service.py [hunk 2]
```diff
@@ -87,6 +87,7 @@ def list_models(self, dataset_id):
         'id': model_full_id,
         'name': model.model_id,
         'datasetId': dataset_id,
+        'type': 'MODEL'
```
Comment (cbwilkes) [inline jupyterlab_bigquery/jupyterlab_bigquery/list_items_handler/service.py:90]: Let's make this a const at the top near SCOPE:

```
MODEL = 'MODEL'
```

File: jupyterlab_bigquery/src/utils/starter_queries.ts (hunk 1 above)
Comment (CynthiaXJia) [reply jupyterlab_bigquery/src/utils/starter_queries.ts]: Sounds good!

File: jupyterlab_bigquery/jupyterlab_bigquery/list_items_handler/service.py (hunk 2 above)
Comment (CynthiaXJia) [reply jupyterlab_bigquery/jupyterlab_bigquery/list_items_handler/service.py]: Got it

File: jupyterlab_bigquery/src/utils/starter_queries.ts [hunk 3]
```diff
@@ -0,0 +1,10 @@
+type QueryType = 'MODEL' | 'TABLE' | 'VIEW';
+
+export function getStarterQuery(type: QueryType, resourceId: string) {
+  console.log('type: ', type);
```
Comment (cbwilkes) [inline jupyterlab_bigquery/src/utils/starter_queries.ts:4]: Can this `console.log` be removed?

Comment (CynthiaXJia) [reply jupyterlab_bigquery/src/utils/starter_queries.ts:4]: Ooops, yes!

**Comment by cbwilkes** [comment]
Can you fix the merge conflicts?

**Pull request merged**